repo/packages/W/weavescope-probe/2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope-probe",
+  "version": "1.2.1",
```

Comment:
If we were use `latests` here along with `latests_release` tag for the image, would it be okay or it's strongly preferred to have explicit versions wherever possible?